### PR TITLE
Update SHOW STATUS docs

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Update the docs for `SHOW STATUS`

This is mostly for https://github.com/pingcap/tidb/pull/48152 but it looks like we never properly documented the existing status variables.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

https://github.com/pingcap/tidb/pull/48152

### CODE REVIEW

**Comment by dveeden** [comment]
image

Source: https://kennytm.github.io/website-docs/dist/

**Comment by yahonda** [comment]
`tiup playground nightly` has not include the bee28760d859ded136806927571a1594b6234c59 yet. Let me review this pull request coming Monday.

**Comment by dveeden** [comment]
> `tiup playground nightly` has not include the bee28760d859ded136806927571a1594b6234c59 yet. Let me review this pull request coming Monday.

Thanks.

Note that if you want you can checkout `pingcap/tidb` from git and switch to the master branch and run `make server && ./bin/tidb-server`. This gives you a running TiDB server with the unistore (instead of TiKV) that you can connect to on port 4000. If something hasn't been merged you can use `gh pr checkout 1234` to checkout a specific PR. The `gh` command is the [GitHub commandline](https://cli.github.com)

File: status-variables.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+### Compression_algorithm
+
+- Scope: SESSION
+- Type: String
+- Indicates the compression algorithm that is used for the MySQL Protocol.
+
+### Compression_level
+
+- Scope: SESSION | GLOBAL
```
Comment (yahonda) [inline status-variables.md:30]: I do not see `Compression_level` from ` SHOW GLOBAL STATUS;`

```
% mysql --comments --host 127.0.0.1 --port 4000 -u root --compress
...
mysql> select tidb_version()\G
*************************** 1. row ***************************
tidb_version(): Release Version: v7.6.0-alpha
Edition: Community
Git Commit Hash: 3f94666b4bafd903886b89206ac603d26d9a9e87
Git Branch: heads/refs/tags/v7.6.0-alpha
UTC Build Time: 2023-11-18 14:25:32
GoVersion: go1.21.3
Race Enabled: false
Check Table Before Drop: false
Store: tikv
1 row in set (0.01 sec)
+-----------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------->
| Variable_name         | Value                                                                                                                                                                                                                                              >
+-----------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------->
| Ssl_cipher            | TLS_AES_128_GCM_SHA256                                                                                                                                                                                                                             >
| Ssl_cipher_list       | RC4-SHA:DES-CBC3-SHA:AES128-SHA:AES256-SHA:AES128-SHA256:AES128-GCM-SHA256:AES256-GCM-SHA384:ECDHE-ECDSA-RC4-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-RC4-SHA:ECDHE-RSA-DES-CBC3-SHA:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES256-SHA:E>
| Ssl_server_not_after  | Feb 18 01:23:56 2024 UTC                                                                                                                                                                                                                           >
| Ssl_server_not_before | Nov 20 01:23:56 2023 UTC                                                                                                                                                                                                                           >
| Ssl_verify_mode       | 5                                                                                                                                                                                                                                                  >
| Ssl_version           | TLSv1.3                                                                                                                                                                                                                                            >
| Uptime                | 2592                                                                                                                                                                                                                                               >
| ddl_schema_version    | 50                                                                                                                                                                                                                                                 >
| server_id             | 04ec893d-7eea-4abb-91f5-a572799f4e7f                                                                                                                                                                                                               >
| tidb_gc_last_run_time | 20231120-11:05:56.379 +0900                                                                                                                                                                                                                        >
| tidb_gc_leader_desc   | host:yahondambp.local, pid:11983, start at 2023-11-20 10:23:56.187046 +0900 JST m=+14.986920604                                                                                                                                                    >
| tidb_gc_leader_lease  | 20231120-11:08:56.389 +0900                                                                                                                                                                                                                        >
| tidb_gc_leader_uuid   | 62fa94ed2240001                                                                                                                                                                                                                                    >
| tidb_gc_safe_point    | 20231120-10:55:56.379 +0900                                                                                                                                                                                                                        >
+-----------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------->
(END)
```

Comment (yahonda) [reply status-variables.md:30]: I'm wondering if it should be `Scope: SESSION`.

File: status-variables.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+### ddl_schema_version
+
+- Scope: SESSION | GLOBAL
+- Type: Integer
+- The version of the DDL schema that is used.
+
+### last_plan_binding_update_time <span class="version-mark">New in v5.2.0</span>
+
+- Scope: SESSION | GLOBAL
```
Comment (yahonda) [inline status-variables.md:84]: `last_plan_binding_update_time` only appears at `show session status`.

Comment (yahonda) [reply status-variables.md:84]: I'm wondering if it should be `Scope: SESSION`

**Review by yahonda — COMMENTED** [review]
Left some comments mainly about the scope of variables.

File: status-variables.md (hunk 1 above)
Comment (dveeden) [reply status-variables.md]: Yes correct. I've updated the scope in 5820f655ff976d8e4d9b7324d211133f651e8160 and verified that there are 9 in global scope and 13 in session scope.

```
sql> \pager tail -1
Pager has been set to 'tail -1'.

sql> SHOW GLOBAL STATUS;
9 rows in set (0.0059 sec)

sql> SHOW SESSION STATUS;
13 rows in set (0.0055 sec)

sql> \pager
Pager has been disabled.

sql> SELECT VERSION();
+------------------------------------------+
| VERSION()                                |
+------------------------------------------+
| 8.0.11-TiDB-v7.6.0-alpha-223-g3f94666b4b |
+------------------------------------------+
1 row in set (0.0014 sec)
```

```
$ grep -E 'Scope:.*GLOBAL' status-variables.md | wc -l
9
$ grep -E 'Scope:.*SESSION' status-variables.md | wc -l
13
```

File: status-variables.md (hunk 2 above)
Comment (dveeden) [reply status-variables.md]: Yes correct. I have fixed this in 5820f655ff976d8e4d9b7324d211133f651e8160

**Comment by dveeden** [comment]
@yahonda Could you check again?

**Comment by yahonda** [comment]
Looks good to me.

File: status-variables.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+title: Server Status Variables
+summary: Use status variables to see the system and session status
+---
+
+# Server Status Variables
+
+Most status variables are implemented to be compatible with MySQL. These variables give information about the global status of the server and the status of the current session.
```
Comment (ran-huang) [inline status-variables.md:8]:
```suggestion
Server status variables provide information about the global status of the server and the status of the current session in TiDB. Most of these variables are designed to be compatible with MySQL.
```

File: status-variables.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+The [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) command can be used to retrieve the global status and the [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) command can be used to see the status of the current session.
+
+The [FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) command is also support to aid MySQL compatibility.
```
Comment (ran-huang) [inline status-variables.md:12]:
```suggestion
You can retrieve the global status using the [SHOW GLOBAL STATUS](/sql-statements/sql-statement-show-status.md) command, and the status of the current session using the [SHOW SESSION STATUS](/sql-statements/sql-statement-show-status.md) command. 

Additionally, the [FLUSH STATUS](/sql-statements/sql-statement-flush-status.md) command is supported for MySQL compatibility.
```

File: status-variables.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+## Variable Reference
```
Comment (ran-huang) [inline status-variables.md:14]:
```suggestion
## Variable reference
```

File: status-variables.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+### Compression
```
Comment (ran-huang) [inline status-variables.md:16]: All the variables are read-only, right?

File: status-variables.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+- Scope: SESSION
+- Type: Timestamp
+- The time and date of the last plan binding update.
+
+### server_id
+
+- Scope: SESSION | GLOBAL
+- Type: String
+- UUID of the server
```
Comment (ran-huang) [inline status-variables.md:92]:
```suggestion
- The UUID of the server.
```

File: status-variables.md (hunk 5 above)
Comment (dveeden) [reply status-variables.md:14]: `system-variables.md` is also using `Variable Reference`. Are they both wrong?

Comment (ran-huang) [reply status-variables.md:14]: Yes, they are both wrong here. According to our style guide, we use sentence case for all headings except for H1.

Comment (dveeden) [reply status-variables.md]: Fixed the other one in 37a8e95c3938e2068db927c955fb1a43600bb0ad

**Review by ran-huang — APPROVED** [review]

**Comment by ran-huang** [comment]
@dveeden Is this PR mergeable?

**Comment by dveeden** [comment]
> @dveeden Is this PR mergeable?

@ran-huang Yes

**Comment by qiancai** [comment]
hi @dveeden, would you please help confirm if this PR also applies to release-7.5?

**Comment by dveeden** [comment]
> hi @dveeden, would you please help confirm if this PR also applies to release-7.5?

@qiancai This needs modifications when applied to release-7.5. I'll create a PR for that.

**Pull request merged**